### PR TITLE
Update the webrtc-pc IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2018-03-06 00:06:58
Finished: 2018-03-06 00:09:54


<h2>Failing Jobs</h2>
<ul>

<li>firefox:nightly</li>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25314)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/349450625)

**Review by foolip — APPROVED** [review]
The changes look reasonable. https://pulls.web-platform-tests.org/build/25558 isn't telling me anything useful, so before merging, can you check how https://github.com/w3c/web-platform-tests/blob/master/webrtc/interfaces.https.html is affected by the change, so that at least it's not now completely broken?

**Comment by lukebjerring** [comment]
Currently afflicted by https://github.com/w3c/web-platform-tests/issues/5714

File: webrtc/interfaces.https.html [hunk 1]
```diff
@@ -107,13 +107,20 @@
...
     const idlArray = new IdlArray();
 
+    let webrtcIdl = fetch('/interfaces/webrtc-pc.idl').then(r => r.text());
+    let mediacaptureMainIdl = fetch('/interfaces/mediacapture-main.idl').then(r => r.text());
+
+    idlArray.add_untested_idls(mediacaptureMainIdl, { only: ['MediaStreamConstraints'] });
+    idlArray.add_idls(webrtcIdl);
+
     idlArray.add_untested_idls('interface EventHandler {};');
+    idlArray.add_idls('interface EventTarget {};');
```
Comment (lukebjerring) [inline webrtc/interfaces.https.html]: This is is example missing interface which had poor error messages (https://github.com/w3c/web-platform-tests/issues/10214)

**Review by ahmadmiz — APPROVED** [review]

**Pull request merged**